### PR TITLE
[test] Add service db test

### PR DESCRIPTION
**Pull request by anyj0527**
- Add negative unittest for serviceDB.

### CODE REVIEW

File: tests/daemon/unittest_service_db.cc [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+  try {
+    db.delete_pipeline ("");
+  } catch (const std::exception &e) {
+    g_critical ("Got Exception: %s", e.what ());
+    gotException = 1;
+  }
+  EXPECT_EQ (gotException, 1);
+}
+
```
Comment (songgot) [inline tests/daemon/unittest_service_db.cc:76]: how about remove this line?

File: tests/meson.build [hunk 2]
```diff
@@ -27,6 +27,13 @@ if gtest_dep.found()
       include_directories: nns_ml_agent_incs,
       cpp_args: [daemon_cpp_db_key_prefix_arg, '-DDB_PATH="."']
     )
+
+  test_db_config_args = declare_dependency(compile_args: ['-DDB_PATH="."', daemon_cpp_db_key_prefix_arg])
```
Comment (jaeyun-jung) [inline tests/meson.build:31]: minor code-style: plz fix indent

File: tests/daemon/unittest_service_db.cc [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+/**
+ * @brief Negative test for model. DB is not initialized.
+ */
+TEST (serviceDB, model_n)
```
Comment (jaeyun-jung) [inline tests/daemon/unittest_service_db.cc:207]: For future release report: please separate all tests, more negative case.
(e.g. updateModelDescInvalidVer_n / setModelInvalidPath / ...)

File: tests/daemon/unittest_service_db.cc (hunk 1 above)
Comment (anyj0527) [reply tests/daemon/unittest_service_db.cc:76]: Thanks, I removed it

File: tests/meson.build (hunk 2 above)
Comment (anyj0527) [reply tests/meson.build]: I added a newline.

File: tests/daemon/unittest_service_db.cc (hunk 3 above)
Comment (anyj0527) [reply tests/daemon/unittest_service_db.cc]: I added tcs for every function calls, PTAL.

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**